Dockerfile

```diff
@@ -14,7 +14,12 @@ RUN CGO_ENABLED=0 GOOS=linux GOARCH=$TARGETARCH go build -o /out/linkerd2-proxy-
 
 ## package runtime
 FROM --platform=$TARGETPLATFORM alpine:20210212
-RUN apk add iptables
+RUN apk add iptables libcap
+RUN touch /run/xtables.lock && chmod 0666 /run/xtables.lock
+RUN setcap cap_net_raw,cap_net_admin+eip /sbin/xtables-legacy-multi
 COPY LICENSE /linkerd/LICENSE
 COPY --from=golang /out/linkerd2-proxy-init /usr/local/bin/proxy-init
+RUN setcap cap_net_raw,cap_net_admin+eip /usr/local/bin/proxy-init
```

Comment:
Do we need to set these on the container? I think we're setting the capabilities at a pod level already? (haven't checked if this is necessary so just curious how it is different in this case)